docs/developer-docs/integrations/sns/launching/launch-summary.md

```diff
@@ -0,0 +1,532 @@
+# Stages of an SNS launch
+
+## Overview
+
+A crucial purpose of an SNS launch is to decentralize the control of an SNS and thereby of
+the dapp that the SNS governs.
+
+Thereby, new tokens must be distributed to a large community to ensure
+proper decentralization of the voting power. There are of course many ways to do so.
+
+The current SNS version provides one simple way to achieve this: a developer can hand over their dapp to the NNS and ask it to start a decentralization
+swap for the newly created SNS.
+
+We first explain the decentralization swap and then the steps included in an SNS launch.
+
+## Key concepts
+
+### Decentralization swap
+
+The launch of each SNS includes a separate **decentralization swap canister** that 
+is owned and run by the IC.
+In more detail, it is controlled by the NNS root canister.
+
+* The swap canister is set up at the start with a defined amount of SNS tokens to be
+  distributed publicly.
+
+* During the decentralization swap, participants can send ICP to the swap canister
+  to contribute to the dapp’s funding.
+
+* At the swap's end the collected ICP are “swapped” for the SNS tokens; the
+  participants get staked SNS tokens in the form of SNS neurons and the SNS gets the
+  collected ICP. Each swap participant will receive their portion of the pool of SNS
+  tokens, pro-rated by their share of the overall number of ICP contributed. 
+  For example, if the swap canister initially held 1000 SNS tokens and 500 ICP tokens
+  were collected during the decentralization swap, then the exchange rate would be 2:1
+  and each participant would get 2 SNS tokens for each ICP token they contributed.
+
+## Stages
+Handing over a dapp's control to a newly created SNS proceeds in the following high level steps.
+Note that the NNS community's approval is relevant in two steps (stages 3, 7, and 9).
+
+### 1. Dapp developers choose the initial parameters of the SNS for a dapp
+  As these parameters define not only the token name but also the tokenomics and how the governance
+  will work, this usually requires a lot of preparation and community engagement already
+  (see [here](../tokenomics/sns-checklist.md) for more information).
+
+  What we have at this stage:
+
+#### Table 1
+
+  <table>
+  <tr>
+    <th>SNS Process</th>
+    <th>State</th>
+  </tr>
+  <tr>
+    <td class="light-green-text">Decentralization swap state</td>
+    <td class="light-green-text">Pending</td>
+  </tr>
+   <tr>
+    <td class="light-green-text">NNS Proposal #1</td>
+    <td class="light-green-text">Pending</td>
+  </tr>
+   <tr>
+    <td class="light-green-text">SNS Proposal #1</td>
+    <td class="light-green-text">Pending</td>
+  </tr>
+   <tr>
+    <td class="light-green-text">NNS Proposal #2</td>
+    <td class="light-green-text">Pending</td>
+  </tr>
```

Comment:
Super Nit that I'd like to point out:

The `pending` state of each of these entries in the table reads like they are already submitted/created (that is consistent with the naming of the various lifecycles we use in the NNS & SNS). I'd suggest  replacing `Pending` with `N/A`, `Not Created` or something along those lines.

Reply:
Agreed. Will revise.